client/containers/Pub/PubDocument/pubBody.scss

```diff
@@ -89,6 +89,7 @@ $pub-body-font-size: if-is-export(14px, 20px);
 		margin: 0;
 		> p {
 			margin: 0;
+			display: inline-block;
```

Comment:
I think this should be `inline` rather than `inline-block`. Otherwise, when the p within the list item overflows, it will go to the next line. But there's also just something a little weird about making a p inline, so I wonder if there's another approach as well.

<img width="519" alt="Screen Shot 2022-05-13 at 09 05 52" src="https://user-images.githubusercontent.com/639110/168289711-bf4f0020-a6f9-4855-a242-4854b7f88d44.png">

Reply:
Playing more...I think this positioning should be `inline`, but targeted to `p:first-child`. Our editor enforces that there is always a p as the first child of the list-item, which is maybe not great, since it means you can never get, say, an image to display just to the left of the bullet. And it's hard to create multiple `p` elements within a list, but there are a few scenarios where this can happen and cause unpredictable results if every p is set as inline (or inline-block for that matter).